src/userApps.ts

```diff
@@ -95,26 +101,50 @@ export class UserApps {
             callbackPath,
         };
 
-        // TODO store the signer key in localStorage
-        const signer = ES256KSigner(privateKey.data.array, true);
+        // TODO use expiresIn to make JWT expire after 5 minutes
 
+        const signer = ES256KSigner(privateKey.data.array, true);
         const jwk = await createJWK(publicKey);
 
         const issuer = toDid(jwk);
 
-        // TODO use expiresIn to make JWT expire after 5 minutes
-        const token = await createJWT(payload, { issuer, signer, alg: 'ES256K-R' });
+        const token = (await Message.sign(payload, { did: issuer, signer: signer as any, alg: 'ES256K-R' })).jwt;
+
+        // const token = (await this.signMessage(payload, keyManager, keyManagerLevel)).jwt;
 
         const requests = [token];
         const requestsString = JSON.stringify(requests);
 
+        console.log(token);
+
         if (redirect) {
             window.location.href = `${getSettings().ssoWebsiteOrigin}/login?requests=${requestsString}`;
             return;
         }
+
         return token;
     }
 
+    static async signMessage(
```

Comment:
seems repedative to have this on userApps and user object. Lets try to abstract away in the future

Reply:
this will help us when we split the sdk for externals and id usage